src/components/Home/CountryList.test.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable comma-dangle */
```

Comment:
- [ ] Instead of disabling the linters, I would recommend fixing this with the code to maintain code standards. The linters are given to make sure you are following the code standard. 
![image](https://user-images.githubusercontent.com/22434660/132801337-e7e3a7d1-d9e2-4bac-ade4-6eb6711567b2.png)